extmod/modframebuf.c

```diff
@@ -41,6 +41,8 @@ typedef struct _mp_obj_framebuf_t {
     uint8_t format;
 } mp_obj_framebuf_t;
 
+STATIC const mp_obj_type_t mp_type_framebuf;
```

Comment:
This will need to be wrapped in `#if !MICROPY_ENABLE_DYNRUNTIME` (see modbtree.c for ex).

Reply:
Done